pkg/systemd/overview-cards/profiles-menu-dialog-body.jsx

```diff
@@ -51,7 +51,7 @@ export class ProfilesMenuDialogBody extends React.Component {
             const active = this.props.active_profile == itm.name;
 
             return (
-                <MenuItem itemId={itm.name} key={itm.name} data-value={itm.name} description={itm.description}>
+                <MenuItem itemId={itm.name} key={itm.name} data-value={itm.name} description={itm.description} isDisabled={this.props.isDisabled || false}>
```

Comment:
Why the `|| false`? Does `isDisabled={undefined}` result in the element to be disabled?